src/main/java/com/parasoft/findings/jenkins/coverage/ParasoftCoverageReportScanner.java

```diff
@@ -43,6 +43,8 @@ public class ParasoftCoverageReportScanner extends AgentFileVisitor<ProcessedFil
     private static final String GENERATED_COBERTURA_REPORT_NAME_SUFFIX = "-cobertura.xml";
     private static final String COVERAGE_TAG_START = "<Coverage ";
     private static final String WORKING_DIRECTORY_ATTR_FORMAT = "pipelineBuildWorkingDirectory=\"%s\" ";
+    private static final String XML_EXTENSION = ".xml";
+    private static final String COVERAGE_TAG = "ver";
```

Comment:
This field name is a bit confusing, as `ver` is only an attribute of the `Coverage` element(tag).